metadata-db-app/src/cmr/metadata_db/api/provider.clj

```diff
@@ -39,6 +40,15 @@
      :body (json/generate-string providers)
      :headers rh/json-header}))
 
+(defn- validate-consortiums
+  "Throws error if consortiums contain anything other than alphabet, numbers,
+  underscores and spaces. Spaces are used as delimiters."
+  [consortiums]
+  (when (re-find #"[^A-Za-z0-9_ ]" consortiums)
```

Comment:
Add validation for consortiums consist of only spaces.

Reply:
If you want to take consortiums consisting of only spaces as valid input, I am ok too. But personally, I don't think it is as clean as failing it during validation. It's not important. It's up to you.

Reply:
Thanks. I will leave it as is with the spaces only case. Same with empty string case. I view it differently.